### ISSUE TITLE
App freezes after Start

### ISSUE THREAD

**Issue by MartinSpecht** [calcitem/Sanmill#639]
**Describe the bug**
Fresh install on Xperia Pro-I Android 13 does not start completely. There is a screen with a board and a choice of Starting the game or getting an introduction, but no tap on the screen is working

**To Reproduce**
1. Go to google play and install the game
2. Click on the game app icon on the device
3. Wait for the app starting
4. => game is frozen

**Expected behavior**
Something which makes the app working? At least any menu, tap or anything else like this?

**Screenshots**
![b4a7d783-e7ea-4bd8-a231-966c8fda3d4f](https://user-images.githubusercontent.com/699087/209336278-c9246b70-6ceb-49fb-a458-8368b9fd4bab.jpg)

**Desktop (please complete the following information):**
 - OS: Android
 - Version 13

**Smartphone (please complete the following information):**
 - Device: Sony XPeria Pro-I (XQ-BE52)
 - OS: Android 13
 - Version 61.22.F.0.147

**Additional context**
none

**Comment by calcitem** [comment]
v2.0.19?

**Comment by MartinSpecht** [comment]
> v2.0.19?

Yes

**Comment by calcitem** [comment]
Thank you for your feedback! I used the Android 13 emulator, set the screen resolution to match that of the Xperia PRO-I, and set the language to Deutsch (Deutschland), but failed to reproduce the issue. I'll try another way.

**Comment by calcitem** [comment]
We have just tested the following mobile phones, and they all passed the test. We will think of other ways.

```
SONY Xperia X F5122, Android 7.1.1, 1080x1920, PASS
Huawei Mate RS NEO-AL00, Android 9, 1440x2880, PASS
Google Pixel 4, Android 13, 1080x2280, PASS
VIVO Y67A, Android 6.0, 720x1280, PASS
```

**Comment by calcitem** [comment]
Are you sure you installed it for the first time, not using an old version before, and then upgraded it on Google Play? Or did you install the old version on Google Play before, uninstall it manually, and then install the new version on Google Play? Thanks!

**Comment by calcitem** [comment]
Hi, is your phone set to dark mode?

**Comment by calcitem** [comment]
In the dark mode, I can indeed see that the icons in the white bar above cannot be displayed, I will fix it. But when I click on the lower screen area, there is a response, and I can move forward step by step. Is yours completely unresponsive?

**Comment by MartinSpecht** [comment]
> Hi, is your phone set to dark mode?

Yes, it is

**Comment by calcitem** [comment]
https://github.com/calcitem/Sanmill/actions/runs/3767250664

Can you try this one？I fix it

**Comment by MartinSpecht** [comment]
I have tried and it works. I think I have not caught the invisible buttons when a tapped randomly at the screen. So it was not frozen, the buttons were simply not visible 😁

**Comment by calcitem** [comment]
Thank you for your feedback!  

This problem is related to Dark mode, the color of the button changes, but the background doesn't change with the mode change, so the button can't be seen.  

A few days ago, a user reported a similar problem on the Google Play Store. His description was very simple. I was wondering until I saw the screenshot you sent in this post. I guessed that the cause of the problem was related to the dark mode.  

Thanks!
